FirebaseFirestoreTestingSupport.podspec

```diff
@@ -0,0 +1,63 @@
+Pod::Spec.new do |s|
+  s.name                    = 'FirebaseFirestoreTestingSupport'
+  s.version                 = '7.7.0'
```

Comment:
How about 1.0.0.

I'm not sure we should tie the version to Firebase versioning?

Reply:
Good idea, will do.

Reply:
Moved under FirebaseTestingSupport folder.

Reply:
Hm, just realized that having the podspec in a subfolder will require `scripts/pod_lib_lint.rb` modifications to accommodate it. I'll look into it but we may want to move the podspec back to the root dir temporary to unblock testing.

Reply:
I'm moving the podspec back to the root dir so far as we do some assumptions in `scripts/pod_lib_lint.rb` that make subfolder support not trivial (not impossible, but just requires a bit of refactoring). 

Another thought, as we may offer the testing support to developer, we may want to keep the spec in the root dir. Let's discuss it a bit more and apply changes if needed in the following PRs.